### ISSUE TITLE
Reload only works two times, then doesn't

### ISSUE THREAD

**Issue by liskin** [chrishrb/go-grip#6]
```console
$ go-grip install.md 
🚀 Starting server: http://localhost:6419/install.md
2024/11/21 19:12:06 error watching ./4913: no such file or directory
2024/11/21 19:12:08 error watching ./4913: no such file or directory
2024/11/21 19:12:08 error watching /install.md: no such file or directory
```

When I change `install.md` for the first time, the first error is printed and the page reloads.
When I change it second time, the other two errors are printed and the page reloads.
Any further changes do absolutely nothing at all.

I run this on Debian testing (trixie) amd64. Installed go-grip using ubi, so presumably https://github.com/chrishrb/go-grip/releases/download/v0.3.0/go-grip-v0.3.0-linux-amd64.tar.gz

**Comment by liskin** [comment]
The error seems to come from https://github.com/aarol/reload/blob/fa84935e67846a0128a0dddd29de00af0f8e951c/watch.go#L64 so this might not be your fault at all actually.

Suppose I could try to reproduce and report it there, but I don't know enough go (yet) for this to be a 5-min task 😞

**Comment by chrishrb** [comment]
Should be fixed now with #31 which was released with version 0.5.3. Please check if the error is still reproducible 👍
